### DISCUSSION TITLE
How to create drop down of selection through field or part?

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11049] [Q&A]
How to create drop down of selection through field or part? on admin side of orchard core

![image](https://user-images.githubusercontent.com/5202291/150582095-2da12a8f-1102-4bd5-a2db-c35fea180958.png)

**Comment by Psichorex** [comment]
Do you want a custom content type to have a drop down list of things defined by you when you create it ?

**Reply by saleemhaq** [reply]
Like we add a field in content type so how i add dropdown?

**Reply by Psichorex** [reply]
You can add a Content Picker Field to your Content Type in this case you will be able to have a drop down list of other content items to attach

**Reply by saleemhaq** [reply]
mean you say i create 3 content item of that name

1- Articles
2- Industry News
3- Partners

so above content item like I have the content type of text fields?

**Reply by saleemhaq** [reply]
> You can add a Content Picker Field to your Content Type in this case you will be able to have a drop down list of other content items to attach

I do that in that way but instead of I have the name like Industry News in category selection i receive the ContentItemIds see :

![image](https://user-images.githubusercontent.com/5202291/150599383-816b8286-88d2-4d35-98f4-9166bbd9fee2.png)

on admin side:

![image](https://user-images.githubusercontent.com/5202291/150599563-d962546b-331f-4232-b6eb-5c5667272c53.png)

Please advise I need the name of industry News instead of of ContentItemIds?

**Comment by saleemhaq** [comment]
@Psichorex please help me on above?
